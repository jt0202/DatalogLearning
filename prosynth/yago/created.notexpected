joss_whedon	the_hunt_for_gollum
ray_bradbury	ichibu_to_zenbudive
jonathan_hoefler	gypsy_rizka
tony_mchale	the_monkey
merzbow	the_bronze
