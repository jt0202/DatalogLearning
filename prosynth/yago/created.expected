denis_héroux	naked_massacre
maurice_cloche	la_portatrice_di_pane_1950_film
jacques_doillon	the_hussy
tzang_merwyn_tong	a_wicked_tale
eddie_griffin	dysfunktional_family
lakis_lazopoulos	fovou_tous_ellines
becky_stark	califunya
françois_dupeyron	the_officers_ward_film
jacques_feyder	crainquebille
rodney_dangerfield	rover_dangerfield
saeed_akhtar_mirza	mohan_joshi_hazir_ho
bert_haanstra	fanfare_film
pieter_verhoeff	mates_film
king_vidor	our_daily_bread_1934_film
leonid_trauberg	the_return_of_maxim
román_viñoly_barreto	la_bestia_debe_morir
jeanpierre_mocky	un_drôle_de_paroissien
eddie_murphy	beverly_hills_cop_ii
norman_mailer	maidstone_film
emilio_fernández	pueblerina
serge_gainsbourg	je_taime_moi_non_plus_film
emilio_fernández	siempre_tuya
king_vidor	better_times_1919_film
rodney_dangerfield	back_by_midnight
luis_bayón_herrera	buenos_aires_a_la_vista
joe_ranft	a_bugs_life
shuji_kataoka	sm_hunter
artie_lange	artie_langes_beer_league
ali_selim	sweet_land
noboru_iguchi	oira_sukeban
márton_keleti	two_confessions
jill_hennessy	the_acting_class
kamal_haasan	manmadhan_ambu
witold_leszczyński	matthews_days
norodom_sihanouk	an_ambition_reduced_to_ashes
vic_morrow	a_man_called_sledge
victor_salva	clownhouse
tamer_hosny	omar__salma_2
leonid_trauberg	alone_1931_film
jacques_feyder	le_grand_jeu_1934_film
ishirō_honda	godzilla_1954_film
marco_ferreri	i_love_you_1986_film
emilio_fernández	rosanna_film
isabel_coixet	map_of_the_sounds_of_tokyo
robin_williams	mrs_doubtfire
eric_byler	americanese
norodom_sihanouk	my_village_at_sunset
chris_rock	good_hair
tony_bui	three_seasons
abel_gance	les_gaz_mortels
eddie_murphy	another_48_hrs
david_mamet	house_of_games
alan_rudolph	afterglow_film
alan_rudolph	love_at_large
kenneth_branagh	loves_labours_lost_2000_film
saeed_akhtar_mirza	arvind_desai_ki_ajeeb_dastaan
jennifer_westfeldt	kissing_jessica_stein
tage_danielsson	the_adventures_of_picasso
alexandre_astruc	the_crimson_curtain
noboru_iguchi	robogeisha
m_night_shyamalan	praying_with_anger
abel_gance	the_mask_of_horror
quentin_tarantino	grave_danger
rick_rubin	tougher_than_leather_film
abel_gance	the_zone_of_death
jamil_dehlavi	the_blood_of_hussain
guan_hu	dirt_1994_film
vilsoni_hereniko	the_land_has_eyes
kenneth_branagh	a_midwinters_tale
leander_haußmann	sonnenallee
w_c_fields	the_bank_dick
abel_gance	jaccuse_1938_film
andrew_lane	jake_speed
alex_chandon	drillbit_film
abel_gance	la_folie_du_docteur_tube
quentin_tarantino	from_dusk_till_dawn
ward_kimball	man_in_space
yılmaz_güney	zavallılar
michel_ocelot	princes_et_princesses
pierre_andre	jangan_pandang_belakang
marco_ferreri	el_cochecito
nelson_george	life_support_film
abel_gance	barberousse
henrigeorges_clouzot	manon_film
kamal_haasan	thevar_magan
abel_gance	un_drame_au_château_dacre
michael_dowse	fubar_2_film
william_gillette	sherlock_holmes_1916_film
michel_houellebecq	la_possibilité_dune_île_film
king_vidor	love_never_dies_1921_film
iain_softley	backbeat_film
henrigeorges_clouzot	le_corbeau
alan_rudolph	trixie_film
alan_rudolph	trouble_in_mind_film
marco_ferreri	bye_bye_monkey
ltÉric_Rochantgt	the_patriots_film
brian_dannelly	saved
john_cleese	romance_with_a_double_bass
serge_gainsbourg	ltÉquateur_filmgt
alan_alda	war_of_nerves_mash
marco_ferreri	break_up_film
alan_rudolph	choose_me
lucien_hubbard	rosemarie_1928_film
jacques_doillon	la_vengeance_dune_femme
kamal_haasan	dasavathaaram
jeanpierre_bekolo	les_saignantes
lucas_belvaux	la_raison_du_plus_faible
king_vidor	his_hour
tamer_hosny	noor_eieny
victor_salva	jeepers_creepers_2
john_cleese	the_human_face
marco_ferreri	the_story_of_piera
clive_barker	hellraiser
alex_chandon	chainsaw_scumfuck
manish_gupta	the_stoneman_murders
peter_sollett	raising_victor_vargas
emma_thompson	wit_film
kevin_munroe	teenage_mutant_ninja_turtles_film_series
dominique_deruddere	everybodys_famous
emilio_fernández	flor_silvestre
abel_gance	le_périscope
lucien_hubbard	the_mysterious_island_1929_film
michael_dowse	kids_in_america_2011_film
sam_shepard	dont_come_knocking
bruce_willis	live_free_or_die_hard
maurice_cloche	peppino_e_violetta
lotte_reiniger	the_adventures_of_prince_achmed
r_kannan	jayamkondaan
frank_d_gilroy	desperate_characters
benito_alazraki	santo_contra_los_zombis
jacques_feyder	faces_of_children
abel_gance	end_of_the_world_1931_film
saeed_akhtar_mirza	salim_langde_pe_mat_ro
michele_lupo	the_master_touch
max_jacoby	dust_2009_film
henrigeorges_clouzot	la_vérité_film
joe_mcstravick	murphy_771
jacques_doillon	the_pirate_1984_film
raymond_bernard	les_misérables_1934_film
emilio_fernández	maclovia_1948_film
tom_waits	big_time_1988_film
isabel_coixet	the_secret_life_of_words
alex_chandon	pervirella
param_gill	hotel_hollywood
zak_penn	the_grand_film
warren_beatty	reds_film
joss_whedon	goners
henrigeorges_clouzot	the_wages_of_fear
isabel_coixet	my_life_without_me
abel_gance	the_ironmaster
michael_dowse	its_all_gone_pete_tong
bill_condon	dreamgirls_film
abel_gance	au_secours
steven_e_de_souza	possessed_2000_film
emilio_fernández	la_choca
abel_gance	the_tenth_symphony
abel_gance	louise_film
marzieh_meshkini	stray_dogs_film
lucas_belvaux	rapt_2009_film
frank_buck_animal_collector	wild_cargo_1934_film
eddie_murphy	coming_to_america
maija_digiorgio	bitter_jester
nasser_taghvai	captain_khorshid
tonia_marketaki	crystal_nights
warren_beatty	bulworth
clive_barker	lord_of_illusions
darren_dalton	the_day_the_earth_stopped
sheldon_reynolds_producer	killers_carnival
dave_mckean	mirrormask
kenneth_branagh	henry_v_1989_film
laura_kirk	lisa_picard_is_famous
dani_levy	alles_auf_zucker
king_vidor	the_turn_in_the_road
john_cleese	wine_for_the_confused
king_vidor	wild_oranges
thomas_gottschalk	die_supernasen
denis_leary	rescue_me_tv_series
jacques_feyder	carnival_in_flanders_film
jaromil_jireš	the_cry_1963_film
karl_hartl	mozart_1955_film
françois_dupeyron	monsieur_ibrahim
marshall_brickman	the_manhattan_project_film
emilio_fernández	the_torch_film
alan_alda	sweet_liberty
jimmy_kimmel	windy_city_heat
marco_ferreri	the_ape_woman
jacques_feyder	latlantide_1921_film
marco_ferreri	ludienza
marty_feldman	in_god_we_trut
cliff_robertson	j_w_coop
jacques_doillon	ponette
steven_e_de_souza	arnolds_wrecking_co
yılmaz_güney	yol
king_vidor	three_wise_fools_1923_film
bert_haanstra	alleman_film
leonid_trauberg	the_vyborg_side
emma_thompson	nanny_mcphee
paul_auster	blue_in_the_face
fernando_cerchio	lulu_1953_film
neil_gaiman	a_short_film_about_john_bolton
giuseppe_colizzi	all_the_way_boys
david_e_durston	i_drink_your_blood
nasser_taghvai	tales_of_kish
ltŠarūnas_Bartasgt	few_of_us
